recycler-connect/recyclers
# Remove recyclers list from rendering on initial page load

## Description
This change guides users to input data to match with recylers before the recycler list displays.

## Related Issue
closes 24


## Updates

### Before
<img width="879" alt="image" src="https://github.com/recycler-connect/recyclers/assets/104444969/bcd2854a-bf71-4332-8bc3-a01a9b7966d0">

### After
<img width="907" alt="image" src="https://github.com/recycler-connect/recyclers/assets/104444969/728e263d-cb81-478f-a21c-1b2c6d4c2eb8">


## Acceptance Criteria

- [x] results list is empty on initial page load
- [x] recycler cards display after, and only after, the materials form is submitted 


## Testing Steps / QA Criteria


1. Visit web app and inspect page. You should not see any matching recylcer facilites until after material data is entered and 'get matching recyclers' button is clicked.

2. Enter material data, click 'get matching recyclers' button, and inspect page. You should now see matching recyclers along with a note directing the user to click on the cards to link to the recycler websites.
